app/src/androidTest/java/ch/hikemate/app/ui/profile/DeleteAccountScreenTest.kt

```diff
@@ -0,0 +1,94 @@
+package ch.hikemate.app.ui.profile
+
+import androidx.compose.ui.test.assertIsDisplayed
+import androidx.compose.ui.test.junit4.createComposeRule
+import androidx.compose.ui.test.onNodeWithTag
+import androidx.compose.ui.test.performClick
+import androidx.compose.ui.test.performTextInput
+import ch.hikemate.app.model.authentication.AuthViewModel
+import ch.hikemate.app.ui.components.BackButton
+import ch.hikemate.app.ui.navigation.NavigationActions
+import com.kaspersky.kaspresso.testcases.api.testcase.TestCase
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.verify
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+
+class DeleteAccountScreenTest : TestCase() {
+  private lateinit var authViewModel: AuthViewModel
+  private lateinit var navigationActions: NavigationActions
+
+  @get:Rule val composeTestRule = createComposeRule()
+
+  @Before
+  fun setUp() {
+    navigationActions = mockk(relaxed = true)
+    authViewModel = mockk(relaxed = true)
+
+    every { navigationActions.goBack() } returns Unit
```

Comment:
Question: Why is this needed? Is it an issue if we go back?

Reply:
The issue is because as we set the content of the test activity to be our `DeleteAccountScreen`, there are no previous page. Going back would then throw an error